crates/cheatcodes/spec/src/vm.rs

```diff
@@ -1077,6 +1077,22 @@ interface Vm {
     #[cheatcode(group = String)]
     function parseBool(string calldata stringifiedValue) external pure returns (bool parsedValue);
 
+    /// Converts the given `string` value to Lowercase.
+    #[cheatcode(group = String)]
+    function toLowercase(string calldata stringifiedValue) external pure returns (string memory stringifiedValue);
```

Comment:
I tested in forge-std to confirm, and this will error with `identifier already declared` in solidity since the `stringifiedValue` name is reused. Suggested var names for all sigs:
```suggestion
    function toLowercase(string calldata input) external pure returns (string memory output);
```